### PR TITLE
UP-46: Layout Product Inner Page

### PR DESCRIPTION
**Pull request by ian-itworkserv**

### CODE REVIEW

File: public/css/home.css [hunk 1]
```diff
@@ -1,18 +1,5 @@
...
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
```
Comment (repoitworkserv) [inline public/css/home.css:2]: @ian-itworkserv  what are the changes at this part?

File: public/css/home.css [hunk 2]
```diff
@@ -24,8 +11,7 @@ time, mark, audio, video {
 
 /* HTML5 display-role reset for older browsers */
 
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
```
Comment (repoitworkserv) [inline public/css/home.css:14]: @ian-itworkserv also on this line?

File: public/css/home.css [hunk 3]
```diff
@@ -1544,10 +1527,53 @@ a.btn.white {
 
 /* quantity select */
 
-.option-list .color-area .quantity-select {
+#quantity_id {
     display: flex;
-    place-content: center;
-    place-items: center;
+    float: right;
```
Comment (repoitworkserv) [inline public/css/home.css:1532]: :+1:

**Comment by ian-itworkserv** [comment]
UP-46

**Review by repoitworkserv — APPROVED** [review]

**Comment by repoitworkserv** [comment]
@ian-itworkserv 
PR Approved
update local branch
- pull from develop then commit to this branch to resolve conflict

File: public/css/home.css (hunk 1 above)
Comment (ian-itworkserv) [reply public/css/home.css:2]: I have a CSS Formatter in my vscode.
It just formats the CSS. That's all.

File: public/css/home.css (hunk 2 above)
Comment (ian-itworkserv) [reply public/css/home.css:14]: The css formatter formats the css file.

**Comment by ian-itworkserv** [comment]
> @ian-itworkserv
> PR Approved
> update local branch
> 
> * pull from develop then commit to this branch to resolve conflict

Kindly check

**Pull request merged**